Specs/DateTime/Spanish/DateTimeModel.json

```diff
@@ -554,7 +554,7 @@
     "Context": {
       "ReferenceDateTime": "2019-08-23T01:00:00"
     },
-    "NotSupported": "javascript, python",
+    "NotSupported": "java, javascript, python",
```

Comment:
Why have these stopped working in Java? Because it misses the new merger method?

Reply:
If so, let's open a Java issue to port this change there and re-enable these. Lower priority than continuing improving Spanish support in .NET for now.

Reply:
yes, I think so.
Ok, will open the issue